include/grpcpp/impl/codegen/callback_common.h

```diff
@@ -94,7 +94,10 @@ class CallbackWithStatusTag
   void Run(bool ok) {
     void* ignored = ops_;
 
-    GPR_CODEGEN_ASSERT(ops_->FinalizeResult(&ignored, &ok));
+    if (!ops_->FinalizeResult(&ignored, &ok)) {
+      // The tag was swallowed
+      return;
+    }
```

Comment:
So in this case what takes care of the unref'ing of the call and the emptying of the arena items?

Reply:
so since the tag was swallowed due to interception. Then when interceptors are done running it would finally result in CallOpSet calling grpc_call_start_batch with 0 ops which should result in the function being run again through the callback cq. Then when FinalizeResult is run again, it will return true this time, and the callback will run. This would be followed by unreffing as normal.

Reply:
This would need restricting CallbackWithStatus to only accept CallOpSetInterface. Would that be a problem?